prep-css-layout-classes/index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Layout Classes</title>
+    <link rel="stylesheet" href="reset.css" />
+    <style>
+
+      /* stylelint-disable block-no-empty */
+
+      body {
+        font-family: sans-serif;
+      }
+
+      h1 {
+        text-align: center;
+      }
+
+      .container {
+        width: 100%;
+        max-width: 992px;
+        margin: auto;
+      }
+
+      .row {
+        width: 100%
```

Comment:
It's possible that you added `float: left` due to this `.row` rule set not quite working. Try adding a `;` semicolon at the end of your `width: 100%` declaration, that should allow your `display: flex;` to work properly 👍